src/base/Components.ts

```diff
@@ -1,108 +1,109 @@
 /**
- * Manages all of the components within the Form.io renderer.
- */
+* Manages all of the components within the Form.io renderer.
```

Comment:
Seemed to mess up the comment spacing..... It should be
```
/**
 *
 */
```

Reply:
Or is my assumption wrong about block comments?

Reply:
that was done automatically by vs code, we can change it if needed